### ISSUE TITLE
Some mypy errors are ignored

### ISSUE THREAD

**Issue by xiongchiamiov** [tsuyoshicho/action-mypy#57]
I have a mypy error that looks like this:

```
importer/flows/helpers/__init__.py: error: Duplicate module named "helpers" (also at "<snip>")
```

Currently this gets ignored because it doesn't have the line and column numbers that `-efm="%f:%l:%c: %t%*[^:]: %m"` expects.

Tentatively https://github.com/winnielabs/action-mypy/commit/42d48176d9ec3333b0d78952e070471ba6d25882 fixes this, but I haven't really tested this thoroughly to see what other effects that has.

I'm suspecting that if mypy gets json output (as per #38), then that would fix this as well.  But the upstream PR is a year old now and there's no decision-making from the core mypy team about what would be needed for it to get merged, so I'm not hopeful about that happening soon.

**Comment by tsuyoshicho** [comment]
Thank you report

I think that, please wait.

**Comment by tsuyoshicho** [comment]
@xiongchiamiov san

Total errror format fix PR #58 is merged.

I have split it into multiple parts to make it easier to read.
Please try the latest.

Thank you for the information.
